beacon-chain/core/peerdas/helpers.go

```diff
@@ -208,28 +208,16 @@ func DataColumnSidecarsForReconstruct(
 	blobKzgCommitments [][]byte,
 	signedBlockHeader *ethpb.SignedBeaconBlockHeader,
 	kzgCommitmentsInclusionProof [][]byte,
-	blobs []cKzg4844.Blob,
+	cells [][cKzg4844.CellsPerExtBlob]cKzg4844.Cell,
+	proofs [][cKzg4844.CellsPerExtBlob]cKzg4844.KZGProof,
```

Comment:
Might make sense to create a new struct here to make this easier on the eye:

```golang
type CellsAndProofs struct {
    cells [cKzg4844.CellsPerExtBlob]cKzg4844.Cell
    proofs [cKzg4844.CellsPerExtBlob]cKzg4844.KZGProof
}
```

Then the function would accept:

```golang
cellsAndProofs []CellsAndProofs
```

Reply:
Added this ([ad5579e](https://github.com/prysmaticlabs/prysm/pull/14160/commits/ad5579efd07646a1366a8394e1ada53d8980a1ef)